src/utils/fixed-forward-ref.ts

```diff
@@ -1,8 +1,7 @@
 import React from 'react'
 
-// eslint-disable-next-line @typescript-eslint/ban-types
 export function fixedForwardRef<T, P = {}>(
-  render: (props: P, ref: React.Ref<T>) => React.ReactNode
+  render: (props: React.PropsWithoutRef<P>, ref: React.Ref<T>) => React.ReactNode
```

Comment:
Was this one intentional?

Reply:
Yeah, it was because without it, i was getting the type error below. I am quite sure what caused this, though, since nothing really changed except removing that linting rule. 
<img width="829" alt="image" src="https://github.com/user-attachments/assets/96307f52-c715-40cf-b9b6-3615fdd4c09e">